<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Identifying Functionally Related Genes</title>
    <meta name="description" content="This is my lab notebook for my undergraduate honors thesis in Dr. Paola Vera-Licona's Computational Systems Medicine Research Group.
">

    <link rel="stylesheet" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/css/main.css">
    <link rel="canonical" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_projects/project1/2019/04/08/functionally-related-genes.html">

<!-- meta robots -->
    
     	<meta name="robots" content="noindex" />
    

<!-- loading fonts -->
    <!-- Open Sans -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/open_sans/style.css" rel="stylesheet">

    <!-- own icon font, created via https://icomoon.io/  -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/lablogIcons/style.css" rel="stylesheet">

    <!-- Ubuntu Mono for code -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/Ubuntu_Mono/style.css" rel='stylesheet' type='text/css'>

<!-- mathjax.org for math equations -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
    imageFont: null,
    extensions: ["tex2jax.js"],
    jax: ["input/TeX","output/HTML-CSS"],
    tex2jax: {inlineMath: [["$","$"],["\\(","\\)"]]} ,
    menuSettings: { context: "Browser" }
    });
    </script>

    <script type="text/javascript"  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>

<!-- highlight.js for code syntax highlighting-->
<link rel="stylesheet" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/highlight/styles/github.css">
<script src="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!--<link rel="stylesheet" href="http://yandex.st/highlightjs/7.3/styles/default.min.css">
<script src="http://yandex.st/highlightjs/7.3/highlight.min.js"></script>
<script>hljs.initHighlightingOnLoad();</script>-->


</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/gastonguay_compsysmed_labnotebook/index.html">Maddie Gastonguay lab notebook</a>

  <nav  class="main-menu">
  <input type="checkbox" id="menu" />
  <label for="menu"></label>
  <div class="trigger">

    <a class="page-link" href="/gastonguay_compsysmed_labnotebook/index.html"><span style = "font-family: 'lablogIcons';">  &#xe605; </span> Maddie Gastonguay lab notebook</a>

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/tasks.html"><span style = "font-family: 'lablogIcons';">  &#xf046; </span> Tasks</a>
    

    
<a class="page-link" href="/gastonguay_compsysmed_labnotebook/tags.html"><span style = "font-family: 'lablogIcons';">  &#xe600; </span> Tags</a>
    

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/archive.html"><span style = "font-family: 'lablogIcons';">  &#xe602; </span> Archive</a>
    

    <br>

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/about.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        About
      </a>
    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/_projects/project1/project01.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        Preliminary Network For Breast Cancer Reversion
      </a>
    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/_projects/project2/project02.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        Cancer Reversion in Claudin Low Triple Negative Breast Cancer
      </a>
    

    

    <br>

    
    <div class="menu-content">
      

    </div>
    

    

  </div>

  </nav>


  </div>


</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Identifying Functionally Related Genes</h1>
    <p class="post-meta">Apr 8, 2019 • maddie</p>

    
    <div>
     
       <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
     
       <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
     
    </div>
    

  </header>

  <article class="post-content">
    <h2 id="filtering-expressed-genes">Filtering Expressed Genes</h2>
<p>RNA-seq data from <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4533-0">here</a>
We are using the list of unfiltered DEGs as the list of all <a href="https://drive.google.com/file/d/1ihB95WGH6Qw9jnLh1zQKSOHgcaoHNePh/view?usp=sharing">expressed genes</a> in MDA-MB-231 because if they have a calculated fold change, they had enough reads to be considered expressed.</p>

<p>Filtering via CNV loss data from <a href="https://cansar.icr.ac.uk/cansar/cell-lines/MDA-MB-231/copy_number_variation/loss/">canSAR</a> webpage. If the gene is listed as having a CNV loss, we removed it from the list of genes.</p>

<p>However, we found that the list of expressed genes didn’t include any genes with CNV loss because list of genes in CNloss file are either not in the raw data, or have zeros for all runs.</p>
<ul>
  <li>ex: DAZ, USP9Y, IFNA5 have zeros for all 8 reads in <a href="https://drive.google.com/file/d/1YR5rj8VQLB0Kun2ivURzbOR8ahXApBnH/view?usp=sharing">raw data.</a></li>
</ul>

<p>We are also filtering by <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868673/">methylation and acetylation data</a>. Inclusion criteria is that genes have both a methylation AND acetylation fold enhancement value whose absolute value is greater than or equal to 2. If this proves to be too stringent, we will use the absolute value of the methylation OR acetylation fold enhancement greater than or equal to 2.</p>

<p>This filtering resulted in <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/expressed_openregions_6784.txt">6783</a> <em>unique</em> genes.</p>

<div style="text-align:center">

  <p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\Venn_allgenes_vs_chromatin_vs_CNloss.png" alt="alt text" /></p>

</div>

<h2 id="calculating-z-scores">Calculating z-scores</h2>
<p>After filtering the list of expressed genes for FDR&lt;0.05 and P-value&lt;0.001, we will try two ways of ranked DEGs when running BiNOM:</p>
<ol>
  <li>
    <p>From <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/rankedDEGs_FDR05_Pval001.txt">lowest to highest p-value</a></p>
  </li>
  <li>
    <p>From <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/rankedDEGs_FDR05_Pval001_rankedzscore.txt">highest to lowest z-score</a></p>
  </li>
</ol>

<p>The Z-score was calculated according to the equation below where $e_{ca}$ is the average expression of a gene in the cancerous cell line, $\mu_N$ is the average expression of the same gene in the normal cell line, and $\sigma_N$ is the standard deviation of the 4 gene expression measurements in the normal cell line. The absolute value of the Z-score was used when ranking lists.</p>

<p align="center">
	$Z = \frac{e_{ca}-\mu_N}{\sigma_N}$
</p>

<p>This method was taken from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4160810/">Rathi et al., 2015</a>. The Z-score reflects the cancerous expression level compared to the distribution of normal samples, and is related to the probability that the expression in the tumor is from the same distribution as is in the normal tissue.</p>

<p>The R code used to calculate the Z-scores is shown below:</p>

<div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="n">df1</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">read.csv</span><span class="p">(</span><span class="s2">"231_expressed_openchromatin_6783.csv"</span><span class="p">)</span><span class="w">
</span><span class="c1">#Create data table with only normalized data for MCF10a</span><span class="w">
</span><span class="n">df</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">df1</span><span class="p">[,</span><span class="nf">c</span><span class="p">(</span><span class="m">3</span><span class="p">,</span><span class="m">21</span><span class="o">:</span><span class="m">24</span><span class="p">)]</span><span class="w">

</span><span class="n">df</span><span class="o">$</span><span class="n">SD</span><span class="o">&lt;-</span><span class="w"> </span><span class="n">apply</span><span class="p">(</span><span class="n">df</span><span class="p">[,</span><span class="m">-1</span><span class="p">],</span><span class="w"> </span><span class="m">1</span><span class="p">,</span><span class="w"> </span><span class="n">sd</span><span class="p">)</span><span class="w">
</span><span class="n">df1</span><span class="o">$</span><span class="n">sd_mcf10a</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">df</span><span class="o">$</span><span class="n">SD</span><span class="w">

</span><span class="n">df1</span><span class="o">$</span><span class="n">zscore</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="p">(</span><span class="n">df1</span><span class="o">$</span><span class="n">MDA_MB.231.mean</span><span class="o">-</span><span class="n">df1</span><span class="o">$</span><span class="n">MCF10A.mean</span><span class="p">)</span><span class="o">/</span><span class="n">df1</span><span class="o">$</span><span class="n">sd_mcf10a</span><span class="w">
</span></code></pre></div></div>

<h2 id="first-order-connectivity">First Order Connectivity</h2>

<h3 id="ranked-by-p-value">Ranked by P-value</h3>

<p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\FOC_analysis_pval.png" alt="P-Value_FOC" /></p>

<h3 id="ranked-by-z-score">Ranked by Z-score</h3>

<p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\FOC_analysis_zscore.png" alt="Zscore_FOC" /></p>

<p><strong>Conclusion:</strong> Move forward with the list of <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/rankedDEGs_FDR05_Pval001_rankedzscore.txt">differentially expressed genes ranked by z-score</a>. This produces a largest connected component of <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/FOC_LCC">30 genes</a>.</p>

<h2 id="second-order-connectivity">Second Order Connectivity</h2>

<p>Moving forward with list of first <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/rankedDEGs_FDR05_Pval001_rankedzscore_top250.txt">250 differentially expressed genes</a> ranked by z-score.</p>

<p>After filtering second order connectivity results for those that are expressed in our data, removing house keeping genes, and then taking the largest connected component, we are left with a list of <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/SOC_LCC_expr_nohkg">75 genes</a>.</p>

<h2 id="weighted-sums">Weighted Sums</h2>

<h3 id="compiling-lists">Compiling Lists</h3>

<p>When calculating weighted sums, we will compare the list of differentially expressed genes, first order connectivity largest component, and second order connectivity largest component to the following three lists:</p>
<ol>
  <li><a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/breast_DO.txt">Breast Cancer Disease Ontology</a> genes associated to breast cancer DO term from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687947/">DOLite</a> in <a href="http://www.bioconductor.org/packages/2.5/bioc/html/GeneAnswers.html">GeneAnswers</a> R package</li>
  <li>The complete <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/HOC.txt">hallmark of cancer list</a> from the <a href="https://acsn.curie.fr/ACSN2/ACSN2.html">atlas of cancer signalling network</a></li>
  <li>Only the genes from list 2 associated with <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/EMT_innateimmune">EMT and innate immune reseponse</a> as these hallmarks are claudin-low specific</li>
</ol>

<h3 id="calculating-weighted-sums">Calculating Weighted Sums</h3>
<pre><code class="language-{r}">weightedsums&lt;-data.frame(gene_set = NA, number_genes = NA, HOC= NA, breastDO= NA, CLgenes=NA, sum=NA)
class(weightedsums$sum)&lt;-'numeric'

varlist&lt;-list(rankedDEGs$X1,FOC$X1,SOC$X1)
for (i in varlist){
  ilen&lt;-length(i)
  x&lt;-length(intersect(i,HOC$X1))/ilen
  y&lt;-length(intersect(i,breastDO$X1))/ilen
  z&lt;-0
  SUM&lt;-((1/3)*x)+((1/3)*y)+((1/3)*z)
  weightedsums[nrow(weightedsums)+1,] = list(NA,ilen,x,y,z,SUM)
}
weightedsums&lt;-weightedsums[-1,]
weightedsums[2,1]&lt;-'DEG'
weightedsums[3,1]&lt;-'FOC'
weightedsums[4,1]&lt;-'SOC'


g1 &lt;- ggplot(data= weightedsums, aes(x = gene_set, y =sum)) +
		geom_bar(stat="identity", aes(fill = gene_set)) +
		ggtitle("Weighted Sums") + xlab("Gene list") +
		ylab("Weighted Sum") + theme(legend.position = "none")
</code></pre>
<p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\weightedsums_plusTABLE.png" alt="weighted sums" /></p>

<p><strong>Conclusion:</strong> Run TRANSFAC and TF analysis in IPA with the <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/SOC_LCC_expr_nohkg">SOC</a> list of 75 genes because it has the highest <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/gene_network/weightedsums_fegm.txt">weighted sum</a>.</p>

  </article>

</div>

<div class="related-posts">
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <h3>Related Posts:</h3>
  <ul>
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/04/05/NetworkAnlysis-final.html">Final Network Analysis
  </a> <span class = "post-meta"> • April  5, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/03/29/NetworkAnlysis3.html">Network Analysis
  </a> <span class = "post-meta"> • March 29, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/01/28/SFA-exploration.html">Exploration of the Uses and Applications of SFA
  </a> <span class = "post-meta"> • January 28, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2019/12/20/SFA-test.html">Preliminary SFA Exploration
  </a> <span class = "post-meta"> • December 20, 2019</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  </ul>
  
</div>



<div class="wrapper">


<div class = "turn" style = " right: 0px" >
 <a rel = "next" href="/gastonguay_compsysmed_labnotebook/_projects/project1/2019/04/10/literature.html"> <span> &rsaquo; </span></a>
</div>





<div class = "turn" style = " left: 0px" >
  <span> &lsaquo; </span>
</div>


</div>


      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Maddie Gastonguay lab notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <li>Maddie Gastonguay lab notebook</li>
          <li><a href="mailto:madeleine.gastonguay@uconn.edu">madeleine.gastonguay@uconn.edu</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/VeraLiconaResearchGroup">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="$grey-color-light" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">VeraLiconaResearchGroup</span>
            </a>
          </li>
          

          


<li>
  <a href="/gastonguay_compsysmed_labnotebook/feed.xml">
    <span class="icon icon-feed">
      <svg viewBox="0 0 16 16">
      <circle cx="3" cy="13" r="2" fill="$grey-color-light" />
        <path fill="$grey-color-light" stroke="#000000" stroke-linecap="round" stroke-linejoin="round" d="M8.5,14.5"/>
        <path fill="$grey-color-light"  d="M8.708,15c-0.552,0-1-0.447-1-1c0-3.147-2.561-5.708-5.708-5.708c-0.552,0-1-0.448-1-1s0.448-1,1-1
          c4.25,0,7.708,3.458,7.708,7.708C9.708,14.553,9.261,15,8.708,15L8.708,15z"/>
        <path fill="$grey-color-light"  d="M13.469,14.999c-0.553,0-1-0.447-1-1C12.469,8.227,7.772,3.531,2,3.531c-0.552,0-1-0.448-1-1s0.448-1,1-1
          c6.875,0,12.469,5.593,12.469,12.468C14.469,14.552,14.021,14.999,13.469,14.999L13.469,14.999z"/>
      </svg>
    </span>
    <span class="feed"> RSS feed </span>
  </a>
</li>



        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">This is my lab notebook for my undergraduate honors thesis in Dr. Paola Vera-Licona's Computational Systems Medicine Research Group.
</p>

      <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>

      <p class="text"><span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Maddie Gastonguay lab notebook</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="http://localhost:4000" property="cc:attributionName" rel="cc:attributionURL">MadeleineGastonguay</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>

      </div>



    </div>

  </div>

</footer>


  </body>

</html>
