---
layout: post
title: "Transcription Factor Analysis"
date: "2019-06-20"
author: maddie
tags:
 - project02
 - static_network
 - bisulfite
 - GeneXplain
 - IPA
 - VEP
 - mutations
---

# Objective
Identify putative transcription factors for our Second Order FunDEG genes. This will be done in two ways: 

1. The MATCH algorithm and TRANSFAC through GeneXplain 
2. Ingenuity Pathway Analysis (IPA) 

The MATCH algorithm uses the TRANSFAC database to identify putative trasncription factor binding sites. IPA analysis identifies TF â€“ FunDEG interactions that have been experimentally validated.

## Using geneXplain Analyze Promoters Workflow

*Note: We modified this workflow to remove the prodution of the table with Entrez IDs*

We ran this geneXplain workflow with 5 different promoter window sizes: **500, 1000, 1500, 2000, and 2500**. 

Each list generated by geneXplain was filtered for a **yes/no ratio > 1.5** and a **p-value < 0.05** (a significance level of 0.001 proved to be too stringent). Then, they were filtered to include only TFs with protein or gene expression data in unmethylated regions according to [bisulfite data](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r110) because we are only interested in those transcription factors that may actually be active in our dataset. Unmethylated regions were identified as cpg sites with a methylation ratio greater than .2, denoting an open chromatin region. There are [8477](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Network_Components/TFs/bisulfite_genes_unmeth_8477.txt) such genes in the bisulfite data for MDA-MB-231.

## Using IPA

To further validate the results of transcription factor analysis using geneXplain, we ran a similar analysis using Ingeneuity Pathway Analysis. We first ran the workflow looking at experimental data from breast cancer samples only, which produced 21 TFs with a p-value less than 0.001. But when these 21 TFs were intersected with the geneXplain lists, there were never more than 2 transcription factors in common. So, we ran IPA again using experimental data from all tissues as well as breast cancer samples. This analysis produced 69 TFs with a p-value less than 0.001, and an overlap with the geneXplain lists ranging from 3-7 transcription factors. 

## Intersection of IPA and geneXplain step :

As mentioned above, we use the intersection of the list of TFs produced by geneXplain and IPA. We computed weighted sums for the geneXplain lists intersected with the IPA list from all tissues plus breast cancer samples. The results are shown below.  

![TF weighted sums]( {{ site.baseurl }}\_assets\images_proj2\TFs_weightedsums_graph_table.png)


Although the list from a promoter window of size 2000 and 2500 have the same weighted sum and size, they actually have different TFs. We decided to compare the coverage of the intersected lists from a winow size of 500, 2000, and 2500 before chosing a list to move forward with.

## Checking Downstream Coverage of Trasncription Factors

$\qquad$ ***How To**: Open the summary file for the AP window size you want to check coverage for in geneXplain. Select the site model IDs that geneXplain predicts the transcription factors bind to. Then click on report on selected matrices in the bar at the top of the geneXplain window. Once the report is generated, sort by the **"total count"** column. So long as there are no rows with 0 counts, you have total coverage of the SOC genes!*

After running this in geneXplain, the three lists we wanted to further investigate (500, 2000, and 2500) **all** cover each of the 316 SOC genes.


## Gain and Loss of TF Binding Sites

As this pipeline uses a multiomics approach, we need to consider how the muatational data for MDA-MB-231 could impact the transcription factors in the network. We will follow the approach outlined in [Lowden et al.](https://europepmc.org/abstract/med/28333948) to identify mutations in regulatory regions that either an increase or decrease in the binding affinity of a TF when compared to the wildtype promoter region. We call those mutations that have an increased affinity for an atypical TF to bind gain of binding site (GOBS) mutations, and those that decrease the affinity of a typical TF loss of binding site (LOBS) mutations. 


![VEP]({{ site.baseurl }}\_assets\images_proj2\VEP.png)

To identify mutations in regulatory regions of our SOC genes, we compared the genes predicted to have mutations in regulatory regions by Ensembl VEP (see image above) to the genes in our SOC set. 

![mutant reg regions]({{ site.baseurl }}\_assets\images_proj2\Venn_mutated_regulatory_regions.png)

This comparison revealed that none of our SOC genes have mutations in their regulatory regions, so we do not have to take this into further consideration.