---
layout: post
title: "ReKINect"
date: "2019-07-07"
author: maddie
tags:
 - project02
 - static_network
 - ReKINect
 - Mutations
---
 

## Objective

Mutations in kinases can have profound effects on signaling pathways in the cell. ReKINect is a computational framework that predicts the likely functionality of mutations in kinase signaling pathways. ReKINect will map mutations to different functional residues and, by doing so, will not only predict how these functional residues might be affected by mutations but also propose a likely functional impact of each mutation to phosphorylation-based signaling networks (Kinases, SH2 proteins and substrates).

A - Essential residues in protein domains (Kinase and SH2 domains).  
B - Determinants of specificity in protein domains (Kinase and SH2 domains).  
C - Phosphorylation sites in any protein.


## Data Processing
ReKINect requires reference AA sequence and mutated AA sequence. To identify said sequences, the [vcf of SNVs](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/231_SNV_correct.vcf) was passed through the variant effect predictor with the ProteinSeqs plugin. This characterizes the effect of each mutation in the [output file](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/231_SNV_correct_output.vcf), and provides the [reference](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/reference.fa) and [mutant](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/mutated.fa) AA sequences of all mutated proteins.

```
vep --input_file 231_SNV_correct.vcf --merged --output_file 231_SNV_correct_output.vcf --plugin ProteinSeqs,reference.fa,mutated.fa --use_given_ref
```

> **Note:** Due  to issues with the strand VEP was reading, some file manipulation was necessary to acheive the correct output. The VCF of SNVs as downloaded from COSMIC was given to VEP ([`231_SNV_COSMIC.vcf`](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/231_SNV_COSMIC.vcf)), but some of the prediceted amino acid changes were incorrect. So, we flipped the amino acids in the VCF to their complement (A --> T, C --> G) to represent what they would be on the other strand ([`231_otherstrand.vcf`](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/231_otherstrand.vcf)). Then we ran VEP on that file and took the entries from each VCF that produced the [correct amino acid ouptut](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/Gene_AA_CORRECT) when VEP id applied to it. This is the [`231_SNV_correct.vcf`](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/231_SNV_correct_output.vcf) file, which was used for the analysis. 

## Run ReKINect
We ran [ReKINect](https://rekinect.science/home) using [complete.fa](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/complete.fa), which has the contents of `reference.fa` and `mutated.fa` in one file. Reference protein sequences are labeled as **>*transcript*_reference** and mutated protein sequences are labeled as **>*transcript*:p.*mutation***, where *transcript* is the Ensmebl ID of the gene, and *mutation* is the amino acid subsitution and location (ex: His499Asn).

### Results
The [ReKINect Output](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project2/Mutations/VEP_runs/ReKINectOutput.txt) is shown below:

![reKINect]({{ site.baseurl }}\_assets\images_proj2\reKINect.png)

None of these mutant Kinases are in our network, but I wanted to determine their effect on elements in our network. The only mutation with existing information is the BRAF mutation, which overactivates MEK. However, MEK is not in our network so I don't need to consider BRAF's effect on the network. Additionaly, EPAHA3 is a damaged transpath protein in the mutant pathways TRANSPATH workflow, so it's effect is already being taken into consideration.

**BRAF:**  Kinase inactivation by hitting essential residue on kinase domain. TKL-RAF-RAF-BRAF protein kinase
p.Gly464Glu
p.Gly504Glu
I'm trying to find consensus on the result of these mutations. A few papers say they make the pathway more active even though the kinase is less active... hmmm

After further literature search, we have decided that the BRAF mutation makes the protein kinase more active, so we will include it in the network as a source node activating MEK.

**ASPM:** mutation around phosphorylation site, but does not destroy site or change function/specificity  
**PKDREJ:** mutation around phosphorylation site, but does not destroy site or change function/specificity  
**STK38:** Mutation around phosphorylation site on kinase protein, but does not destroy site or change function/specificity  
**TNFRSF21:** Mutation around phosphorylation site, but does not destroy site or change function/specificity  


**Uninterpreted mutation on kinase protein (7):**  
ALPK2, EPHA3,LRRK2,TRIO, WNK1, MAP3K19, PDGFRA


## KinomeXplorer
In order to determine kinase signaling changes upstream of mutated proteins, we used KinomeXplorer to identify atypical phosphorylating species for the mutant transcript compared to the wild type. Knowing that the only mutated kinases in the network are TP53 and BRAF whose non-canonically activated pathways will be added to the network, we will use this software to determine nodes that are not in our network but have a gain-of-phosphorylation site mutation that are regulated by members of the network.  

We filtered our vcf file to only include those mutations that change to a Threonine, Serine, or Tyrosine as these are the phosphorylatable amino acids. There were no mutations in our data where a phosphorylation site remained but with a different amino acid. We did not consider loss of phosphorylation sites as these nodes are not in our network so we are only concerned with those that could possibly now interact with members of our network if they hadn't before.

This filter returned two genes with such mutations: TJP1, and MAP7D3. 

We input the protein sequences for each mutation as generated by the VEP ProteinSeqs plugin into NetPhorest. Then we selected the residues that were of interest and NetPhorest predicted what elements would now be able to interact with each of these genes. 

<div style="text-align:center" markdown="1">
![netphorest output]({{ site.baseurl }}\_assets\images\netphorest_output_2.png)

![netphorest output]({{ site.baseurl }}\_assets\images\netphorest_output.png)
</div>

TJP1 had 4 Glycine to Serine mutations at different locations that all lead to the same outcome. However, the highest scoring interaction scored 0.20, and the developers of NetPhorest advise not considering anything that scores below 0.30, so we will not consider it in our network. 

MAP7D3 also had multiple glycine to serine mutations at different locations and NetPhorest predicted two kinase families with a score of 0.37: the PKC group and the CK2 group. However, after a quick literature search we determined that it has little to do with breast cancer so we will not include it in the network.
