---
layout: post
title: "Using Ingenuity Pathways Analysis"
date: "2019-05-13"
author: maddie
tags:
 - project01
 - ISG_workshop
 - HowTo
---

## Using IPA with a data set
- Have a list of genes that are either up regulated or down regulated (may have p-value)
- Want to determine downstream or upstream effects
- Data set has: ID, differential expression, p-value/FDR
- Can look at RNAseq and proteomics side by side
- Can compare your analsyis to other studies


## Using IPA without a data set
- If you're interested in one gene, you can enter the gene name in IPA and it gives you upstream names, downstream genes in an interactive map and you can build a network from there
- You can also search disease name and pull up evey molecule associated with a dsisease
- Public dataset --> use project manager search


- IPA gives direrctionaly relations from findings
- Download raw data from publications and then reprocess it
- Project Manager on left side of screen
	- libraries 
- Search bar on top
	- Make sure you're using the right tab


## How To:
1. Data has columns of: ID (only **required** column), ratio (or log fold change), p-value, adj p-value
	- Data can be excel, tab delimited, or Cuffdiff file
	- Don't usually have to convert IDs (gene symbols or ensembl IDs work)
2. Upload data set into IPA and save it
	- Project Manager > NEW > upload data set
	- Don't need to change settings unless you're using microarray data
		- then need to chose which microarry chip you used
	- Define columns in uploaded file
		- one observation means one comparison! (fold change, p-value, and FDR from one comparison are all Observation 1)
		- ID (specify type), Observation 1 Expr Ratio, Observation 1 Expr pvalue, Observation 1 FDR
		- Select ignore for the columns you don't want
	- If this is the first time you're uploading data, go to Data Summary and see how many gene IDs are being successfully matched
3. Set up th core analysis
	- NEW > Core Analysis from dropdown menu OR Analyze\Filter Dataset > Core Analysis in bottom right corner (they do the same exact thing)
	- Networks Interaction > check **Causal networks**!!!
		- important for MRs
		- can use this result to determine similarities between your results and public data sets
	- Leave the other options as default to start but then can change things
		- Ex: can run analysis only using findings from one type of tissue...
	- Can set Cutoffs such as p-value and fold change to shrink dataset
		- **Recommended:** shrink down RNA-seq to between 1,000 - 2,000 genes (Don't do more than 3,000 or 4,000 because the more you use, the more things will be mapped so p-values won't make sense)
	- Click Run Analysis (usually takes 3-5 minutes)
		- Ends up the in the folder you specify
4. Check out results
	- Click on core analysis results
	- You see multiple tabs with different information from this analysis
	- Canonical Pathways = well known pathways enrichment (similar to KEGG)
		 - Height of the bar corresponds to the significance of the p-value
		 - Orange means the pathway is predicted to be activated
		 - Blue means the pathway is prediceted to be inhibited
		 - Click on a bar to see the genes related to the pathway
		 - **OPEN PATHWAY** button gives you a visualization of pathway maps
	- Can see all the pubmed links for each pathway
		- Right click and then view references
		- Only gives paper names
	- Red and green mean genes are in your data set (red means down up regulated, green means down regulated)
	-Analysis Match
		- Gives you a table of the match score from your own analysis to publicly avaialable data sets
		- Negative CP matching percent meanst that pathways activated in your data set are inactivated in the publicly available data
	- Use **OVERLAY** tool > **MAP (Molecule Activity Predictor)**
		- Red paint bucket = what if this gene is activated
		- Green paint bucket = what if this gene is deactivated
		- Paint the gene of interest and then click **START PREDICTION**
		- Tells you predicted up and down regulation of downstream genes
	- Export Data > All Relationships




