---
layout: post
title: "Investigating Phosphorylation Mutations"
date: "2019-05-15"
author: lauren, maddie
tags:
 - project01
 - static_network
 - mutations
---

# Question:
How do mutations in 5' UTR affect the ability for a TF to bind/ affect gene expression?
  - if you know that a protein is expressed or the mRNA is present then there must be a TF able to bind there
  - does it matter which one? (ie. if JUN is a TF for a SOC gene whose protein and/or mRNA is present but its promoter region is mutated is there another TF in the cell that is accounting for this expression that we haven't accounted for?)

## ReKINect

ReKINect is a computational framework that predicts the likely functionality of mutations. ReKINect will map mutations to different functional residues and, by doing so, will not only predict how these functional residues might be affected by mutations but also propose a likely functional impact of each mutation to phosphorylation-based signaling networks (Kinases, SH2 proteins and substrates).

A - Essential residues in protein domains (Kinase and SH2 domains).  
B - Determinants of specificity in protein domains (Kinase and SH2 domains).  
C - Phosphorylation sites in any protein.


### Data Processing
ReKINect requires reference AA sequence and mutated AA sequence.  
Our list of mutant protein IDs is from [DEPMAP](https://depmap.org/portal/cell_line/MDAMB231_BREAST?tab=mutation). This list contains [569 mutations](https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/MDAMB231_BREAST%20mutations.csv)  

The list of SNPs was passed through the variant effect predictor with the ProteinSeqs plugin. This provides the reference and mutant AA sequences.

```
vep -i 231SNV.vcf -o 231.txt --plugin ProteinSeqs 231ref.fa,231mut.fa --database
```
And then Lauren wrote 5 different scripts to get the correct formatting
<iframe src="https://giphy.com/embed/eBCnpuRGBhQGY" width="480" height="409" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/mondays-eBCnpuRGBhQGY">via GIPHY</a></p>

### ReKINect output
![alt text]({{ site.baseurl }}\_assets\images\ReKINect output.PNG)

**Conclusion:** ReKINect will tell us if we have changes in phosphorylation pathways. That data could be useful for signaling. However, it will not give us information on perhaps a mutant TF.

### THEN
We looked at the output of desktop run VEP, and noticed it was different than GeneXplain VEP. Most notably it did not contain TP53 missense mutations which we **know** is in this tumor.
There appeared to be a discrepancy between sites in DepMAP, COSMIC, and which genome assembly was used.
So everything we was at best inaccurate- but overall useless.


# ROUND 2: Current Workflow with results
**Input:** [COSMIC mutations](https://platform.genexplain.com/bioumlweb/#de=data/Projects/SalazarCortes%20DataAnalysis/Data/Gastonguay/MDAMB231/231SNV_COSMIC%20table%20filtered_framemisssplicedel%20Ensembl)
This time we are starting the the COSMIC data.  
Analyzed using GeneXplain VEP.  
We overlapped the results with all genes expressed in our gene expression data, and the genes in our network.


![alt text]({{ site.baseurl }}\_assets\images\Venn_cosmic_allgenes_SOC.PNG)

1 in overlap=TP53  
[R280K mutation](www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3483014): Disordered growth in 3D cultures, induction of mevalonate pathway genes; induction of migration-related mutant p53 signature genes and migratory phenotype; inhibition of p63 downstream transcriptional program and TGFÎ² -induced metastasis; inhibition of apoptosis mediated by the vit. D receptor, induction of chemokines and other inflammatory mediators (MDA-MB-231).  
pathways mentioned are not in ACSN
## ReKINect
run using complete.fa

**Results**  
32 interpreted mutations  
12 genes

**BRAF:**  Kinase inactivation by hitting essential residue on kinase domain. TKL-RAF-RAF-BRAF protein kinase
p.Gly464Glu
p.Gly504Glu
I'm trying to find consensus on the result of these mutations. A few papers say they make the pathway more active even though the kinase is less active... hmmm

After further literature search, we have decided that the BRAF mutation makes the protein kinase more active, so we will include it in the network as a source node activating MEK.

**ASPM:** mutation around phosphorylation site, but does not destroy site or change function/specificity  
**PKDREJ:** mutation around phosphorylation site, but does not destroy site or change function/specificity  
**STK38:** Mutation around phosphorylation site on kinase protein, but does not destroy site or change function/specificity  
**TNFRSF21:** Mutation around phosphorylation site, but does not destroy site or change function/specificity  


**Uninterpreted mutation on kinase protein (7):**  
ALPK2, EPHA3,LRRK2,TRIO, WNK1, MAP3K19, PDGFRA


# KinomeXplorer
In order to determine kinase signaling changes upstream of mutated proteins, we used KinomeXplorer to identify atypical phosphorylating species for the mutant transcript compared to the wild type. Knowing that the only mutations in the network are NF2 and TP53, which have been taken into consideration, we will use this software to determine nodes that are not in our network but have a gain-of-phosphorylation site mutation that are regulated by members of the network.  

We filtered our vcf file to only include those mutations that change to a Threonine, Serine, or Tyrosine as these are the phosphorylatable amino acids. There were no mutations in our data where a phosphorylation site remained but with a different amino acid. We did not consider loss of phosphorylation sites as these nodes are not in our network so we are only concerned with those that could possibly now interact with members of our network if they hadn't before.

This filter returned two genes with such mutations: TJP1, and MAP7D3. 

We input the protein sequences for each mutation as generated by the VEP ProteinSeqs plugin into NetPhorest. Then we selected the residues that were of interest and NetPhorest predicted what elements would now be able to interact with each of these genes. 

<div style="text-align:center" markdown="1">
![netphorest output]({{ site.baseurl }}\_assets\images\netphorest_output_2.png)

![netphorest output]({{ site.baseurl }}\_assets\images\netphorest_output.png)
</div>

TJP1 had 4 Glycine to Serine mutations at different locations that all lead to the same outcome. However, the highest scoring interaction scored 0.20, and the developers of NetPhorest advise not considering anything that scores below 0.30, so we will not consider it in our network. 

MAP7D3 also had multiple glycine to serine mutations at different locations and NetPhorest predicted two kinase families with a score of 0.37: the PKC group and the CK2 group. However, after a quick literature search we determined that it has little to do with breast cancer so we will not include it in the network.
