<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Investigating Phosphorylation Mutations</title>
    <meta name="description" content="This is my lab notebook for my undergraduate honors thesis in Dr. Paola Vera-Licona's Computational Systems Medicine Research Group.
">

    <link rel="stylesheet" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/css/main.css">
    <link rel="canonical" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_projects/project1/2019/05/15/ReKINect.html">

<!-- meta robots -->
    
     	<meta name="robots" content="noindex" />
    

<!-- loading fonts -->
    <!-- Open Sans -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/open_sans/style.css" rel="stylesheet">

    <!-- own icon font, created via https://icomoon.io/  -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/lablogIcons/style.css" rel="stylesheet">

    <!-- Ubuntu Mono for code -->
    <link href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/Ubuntu_Mono/style.css" rel='stylesheet' type='text/css'>

<!-- mathjax.org for math equations -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
    imageFont: null,
    extensions: ["tex2jax.js"],
    jax: ["input/TeX","output/HTML-CSS"],
    tex2jax: {inlineMath: [["$","$"],["\\(","\\)"]]} ,
    menuSettings: { context: "Browser" }
    });
    </script>

    <script type="text/javascript"  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>

<!-- highlight.js for code syntax highlighting-->
<link rel="stylesheet" href="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/highlight/styles/github.css">
<script src="http://localhost:4000/gastonguay_compsysmed_labnotebook/_assets/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!--<link rel="stylesheet" href="http://yandex.st/highlightjs/7.3/styles/default.min.css">
<script src="http://yandex.st/highlightjs/7.3/highlight.min.js"></script>
<script>hljs.initHighlightingOnLoad();</script>-->


</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/gastonguay_compsysmed_labnotebook/index.html">Maddie Gastonguay lab notebook</a>

  <nav  class="main-menu">
  <input type="checkbox" id="menu" />
  <label for="menu"></label>
  <div class="trigger">

    <a class="page-link" href="/gastonguay_compsysmed_labnotebook/index.html"><span style = "font-family: 'lablogIcons';">  &#xe605; </span> Maddie Gastonguay lab notebook</a>

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/tasks.html"><span style = "font-family: 'lablogIcons';">  &#xf046; </span> Tasks</a>
    

    
<a class="page-link" href="/gastonguay_compsysmed_labnotebook/tags.html"><span style = "font-family: 'lablogIcons';">  &#xe600; </span> Tags</a>
    

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/archive.html"><span style = "font-family: 'lablogIcons';">  &#xe602; </span> Archive</a>
    

    <br>

    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/about.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        About
      </a>
    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/_projects/project1/project01.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        Preliminary Network For Breast Cancer Reversion
      </a>
    
      <a class="page-link" href="/gastonguay_compsysmed_labnotebook/_projects/project2/project02.html">
        <span style = "font-family: 'lablogIcons';">  &#xe60a; </span>
        
        
        Cancer Reversion in Claudin Low Triple Negative Breast Cancer
      </a>
    

    

    <br>

    
    <div class="menu-content">
      

    </div>
    

    

  </div>

  </nav>


  </div>


</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Investigating Phosphorylation Mutations</h1>
    <p class="post-meta">May 15, 2019 • lauren, maddie</p>

    
    <div>
     
       <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
     
       <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
     
       <a href="/gastonguay_compsysmed_labnotebook/tags.html#mutations" title="Pages tagged mutations" rel="tag" class = "tag"> mutations </a>  &nbsp
     
    </div>
    

  </header>

  <article class="post-content">
    <h2 id="rekinect">ReKINect</h2>

<p>ReKINect is a computational framework that predicts the likely functionality of mutations. ReKINect will map mutations to different functional residues and, by doing so, will not only predict how these functional residues might be affected by mutations but also propose a likely functional impact of each mutation to phosphorylation-based signaling networks (Kinases, SH2 proteins and substrates).</p>

<p>A - Essential residues in protein domains (Kinase and SH2 domains).<br />
B - Determinants of specificity in protein domains (Kinase and SH2 domains).<br />
C - Phosphorylation sites in any protein.</p>

<h3 id="data-processing">Data Processing</h3>
<p>ReKINect requires reference AA sequence and mutated AA sequence.<br />
Our list of mutant protein IDs is from <a href="https://depmap.org/portal/cell_line/MDAMB231_BREAST?tab=mutation">DEPMAP</a>. This list contains <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/project1/MDAMB231_BREAST%20mutations.csv">569 mutations</a></p>

<p>The list of SNPs was passed through the variant effect predictor with the ProteinSeqs plugin. This provides the reference and mutant AA sequences.</p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>vep -i 231SNV.vcf -o 231.txt --plugin ProteinSeqs 231ref.fa,231mut.fa --database
</code></pre></div></div>
<p>And then Lauren wrote 5 different scripts to get the correct formatting</p>
<iframe src="https://giphy.com/embed/eBCnpuRGBhQGY" width="480" height="409" frameborder="0" class="giphy-embed" allowfullscreen=""></iframe>
<p><a href="https://giphy.com/gifs/mondays-eBCnpuRGBhQGY">via GIPHY</a></p>

<h3 id="rekinect-output">ReKINect output</h3>
<p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\ReKINect output.PNG" alt="alt text" /></p>

<p><strong>Conclusion:</strong> ReKINect will tell us if we have changes in phosphorylation pathways. That data could be useful for signaling. However, it will not give us information on perhaps a mutant TF.</p>

<h3 id="then">THEN</h3>
<p>We looked at the output of desktop run VEP, and noticed it was different than GeneXplain VEP. Most notably it did not contain TP53 missense mutations which we <strong>know</strong> is in this tumor.
There appeared to be a discrepancy between sites in DepMAP, COSMIC, and which genome assembly was used.
So everything we was at best inaccurate- but overall useless.</p>

<h1 id="round-2-current-workflow-with-results">ROUND 2: Current Workflow with results</h1>
<p><strong>Input:</strong> <a href="https://github.com/VeraLiconaResearchGroup/CancerReversion/blob/master/_projects/geneXplain_backup/old/231SNV_COSMIC%20table%20filtered_framemisssplicedel%20Ensembl.txt">COSMIC mutations</a>
This time we are starting with the COSMIC data.<br />
Analyzed using GeneXplain VEP.<br />
We overlapped the results with all genes expressed in our gene expression data, and the genes in our network.</p>

<p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\Venn_cosmic_allgenes_SOC.PNG" alt="alt text" /></p>

<p>1 in overlap=TP53<br />
<a href="www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3483014">R280K mutation</a>: Disordered growth in 3D cultures, induction of mevalonate pathway genes; induction of migration-related mutant p53 signature genes and migratory phenotype; inhibition of p63 downstream transcriptional program and TGFβ -induced metastasis; inhibition of apoptosis mediated by the vit. D receptor, induction of chemokines and other inflammatory mediators (MDA-MB-231).<br />
pathways mentioned are not in ACSN</p>
<h2 id="rekinect-1">ReKINect</h2>
<p>run using complete.fa</p>

<p><strong>Results</strong><br />
32 interpreted mutations<br />
12 genes</p>

<p><strong>BRAF:</strong>  Kinase inactivation by hitting essential residue on kinase domain. TKL-RAF-RAF-BRAF protein kinase
p.Gly464Glu
p.Gly504Glu
I’m trying to find consensus on the result of these mutations. A few papers say they make the pathway more active even though the kinase is less active… hmmm</p>

<p>After further literature search, we have decided that the BRAF mutation makes the protein kinase more active, so we will include it in the network as a source node activating MEK.</p>

<p><strong>ASPM:</strong> mutation around phosphorylation site, but does not destroy site or change function/specificity<br />
<strong>PKDREJ:</strong> mutation around phosphorylation site, but does not destroy site or change function/specificity<br />
<strong>STK38:</strong> Mutation around phosphorylation site on kinase protein, but does not destroy site or change function/specificity<br />
<strong>TNFRSF21:</strong> Mutation around phosphorylation site, but does not destroy site or change function/specificity</p>

<p><strong>Uninterpreted mutation on kinase protein (7):</strong><br />
ALPK2, EPHA3,LRRK2,TRIO, WNK1, MAP3K19, PDGFRA</p>

<h1 id="kinomexplorer">KinomeXplorer</h1>
<p>In order to determine kinase signaling changes upstream of mutated proteins, we used KinomeXplorer to identify atypical phosphorylating species for the mutant transcript compared to the wild type. Knowing that the only mutations in the network are NF2 and TP53, which have been taken into consideration, we will use this software to determine nodes that are not in our network but have a gain-of-phosphorylation site mutation that are regulated by members of the network.</p>

<p>We filtered our vcf file to only include those mutations that change to a Threonine, Serine, or Tyrosine as these are the phosphorylatable amino acids. There were no mutations in our data where a phosphorylation site remained but with a different amino acid. We did not consider loss of phosphorylation sites as these nodes are not in our network so we are only concerned with those that could possibly now interact with members of our network if they hadn’t before.</p>

<p>This filter returned two genes with such mutations: TJP1, and MAP7D3.</p>

<p>We input the protein sequences for each mutation as generated by the VEP ProteinSeqs plugin into NetPhorest. Then we selected the residues that were of interest and NetPhorest predicted what elements would now be able to interact with each of these genes.</p>

<div style="text-align:center">
  <p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\netphorest_output_2.png" alt="netphorest output" /></p>

  <p><img src="/gastonguay_compsysmed_labnotebook\_assets\images\netphorest_output.png" alt="netphorest output" /></p>
</div>

<p>TJP1 had 4 Glycine to Serine mutations at different locations that all lead to the same outcome. However, the highest scoring interaction scored 0.20, and the developers of NetPhorest advise not considering anything that scores below 0.30, so we will not consider it in our network.</p>

<p>MAP7D3 also had multiple glycine to serine mutations at different locations and NetPhorest predicted two kinase families with a score of 0.37: the PKC group and the CK2 group. However, after a quick literature search we determined that it has little to do with breast cancer so we will not include it in the network.</p>

  </article>

</div>

<div class="related-posts">
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <h3>Related Posts:</h3>
  <ul>
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/04/05/NetworkAnlysis-final.html">Final Network Analysis
  </a> <span class = "post-meta"> • April  5, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#mutations" title="Pages tagged mutations" rel="tag" class = "tag"> mutations </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/03/29/NetworkAnlysis3.html">Network Analysis
  </a> <span class = "post-meta"> • March 29, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#mutations" title="Pages tagged mutations" rel="tag" class = "tag"> mutations </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2020/01/28/SFA-exploration.html">Exploration of the Uses and Applications of SFA
  </a> <span class = "post-meta"> • January 28, 2020</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#mutations" title="Pages tagged mutations" rel="tag" class = "tag"> mutations </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  <li class="relatedPost">
  <a href="/gastonguay_compsysmed_labnotebook/_projects/project2/2019/12/20/SFA-test.html">Preliminary SFA Exploration
  </a> <span class = "post-meta"> • December 20, 2019</span>
  
  <span class = "post-meta"> • </span>
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#project01" title="Pages tagged project01" rel="tag" class = "tag"> project01 </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#static_network" title="Pages tagged static_network" rel="tag" class = "tag"> static_network </a>  &nbsp
  
    <a href="/gastonguay_compsysmed_labnotebook/tags.html#mutations" title="Pages tagged mutations" rel="tag" class = "tag"> mutations </a>  &nbsp
  
  

  </li>
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  
  </ul>
  
</div>



<div class="wrapper">


<div class = "turn" style = " right: 0px" >
 <a rel = "next" href="/gastonguay_compsysmed_labnotebook/_projects/project1/2019/05/16/Planning-Boolean-Network.html"> <span> &rsaquo; </span></a>
</div>





<div class = "turn" style = " left: 0px" >
 <a rel = "prev" href="/gastonguay_compsysmed_labnotebook/_projects/project1/2019/05/13/using-IPA.html"> <span> &lsaquo; </span></a>
</div>



</div>


      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Maddie Gastonguay lab notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <li>Maddie Gastonguay lab notebook</li>
          <li><a href="mailto:madeleine.gastonguay@uconn.edu">madeleine.gastonguay@uconn.edu</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/VeraLiconaResearchGroup">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="$grey-color-light" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">VeraLiconaResearchGroup</span>
            </a>
          </li>
          

          


<li>
  <a href="/gastonguay_compsysmed_labnotebook/feed.xml">
    <span class="icon icon-feed">
      <svg viewBox="0 0 16 16">
      <circle cx="3" cy="13" r="2" fill="$grey-color-light" />
        <path fill="$grey-color-light" stroke="#000000" stroke-linecap="round" stroke-linejoin="round" d="M8.5,14.5"/>
        <path fill="$grey-color-light"  d="M8.708,15c-0.552,0-1-0.447-1-1c0-3.147-2.561-5.708-5.708-5.708c-0.552,0-1-0.448-1-1s0.448-1,1-1
          c4.25,0,7.708,3.458,7.708,7.708C9.708,14.553,9.261,15,8.708,15L8.708,15z"/>
        <path fill="$grey-color-light"  d="M13.469,14.999c-0.553,0-1-0.447-1-1C12.469,8.227,7.772,3.531,2,3.531c-0.552,0-1-0.448-1-1s0.448-1,1-1
          c6.875,0,12.469,5.593,12.469,12.468C14.469,14.552,14.021,14.999,13.469,14.999L13.469,14.999z"/>
      </svg>
    </span>
    <span class="feed"> RSS feed </span>
  </a>
</li>



        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">This is my lab notebook for my undergraduate honors thesis in Dr. Paola Vera-Licona's Computational Systems Medicine Research Group.
</p>

      <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>

      <p class="text"><span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Maddie Gastonguay lab notebook</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="http://localhost:4000" property="cc:attributionName" rel="cc:attributionURL">MadeleineGastonguay</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>

      </div>



    </div>

  </div>

</footer>


  </body>

</html>
